src/OrchardCore.Modules/OrchardCore.Media/Controllers/AdminController.cs

```diff
@@ -457,5 +464,30 @@ public async Task<IActionResult> Options()
 
             return View(_mediaOptions);
         }
+
+        private HashSet<string> GetRequestedExtensions(string exts, bool fallback)
+        {
+            if (!String.IsNullOrWhiteSpace(exts))
+            {
+                var extensions = exts.Split(new char[] { ' ', ',' }, StringSplitOptions.RemoveEmptyEntries | StringSplitOptions.TrimEntries);
+
+                var validExtensions = _mediaOptions.AllowedFileExtensions
+                    .Intersect(extensions)
+                    .ToHashSet(StringComparer.OrdinalIgnoreCase);
+
+                if (validExtensions.Count > 0)
```

Comment:
```suggestion
                if (requestedExtensions.Any())
```

Reply:
`.Count` or `.Length` should be used over `.Any()` when possible.